### PR TITLE
Various Landing Pages & Info Tweaks

### PR DESCRIPTION
**Pull request by kingthorin**
- index.md > Add flagship and twitter badges.
- info.md > Switch to common svg images and remove png references.
- /assets/images > Remove local pngs no longer used.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: info.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 ### Downloads
 
 * [Download OWASP Zap!](https://github.com/zaproxy/zaproxy/wiki/Downloads)
 
-### Code Repository
+### Code Repositories
 
 * [zaproxy](https://github.com/zaproxy/zaproxy/)
 * [zap-extensions](https://github.com/zaproxy/zap-extensions/)
```
Comment (psiinon) [inline info.md]: worth adding the HUD one too?

Comment (kingthorin) [reply info.md]: Sure

**Comment by kingthorin** [comment]
Done

File: index.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ pitch: The OWASP Zed Attack Proxy (ZAP) is one of the world’s most popular fre
 ---
 <!-- rebuild 40 -->
 
+[![OWASP Flagship](https://img.shields.io/badge/owasp-flagship%20project-48A646.svg)](https://www.owasp.org/index.php/OWASP_Project_Inventory#tab=Flagship_Projects)
```
Comment (thc202) [inline index.md]: The links does not seem to be working properly. Should it link to the new site `/projects#flagships` instead? https://www2.owasp.org/projects#flagships

Comment (kingthorin) [reply index.md]: I'll switch it up, unfortunately that link is 100% either but I think that's more of a functional issue. I'll ask about linking to tabs on the new site(s) in slack.

**Comment by kingthorin** [comment]
Done

**Comment by thc202** [comment]
Thanks!

**Pull request merged**